### PR TITLE
update documentation for types

### PR DESCRIPTION
**Pull request by johnDance**

### CODE REVIEW

File: docs/trex_typescript.md [hunk 1]
```diff
@@ -78,37 +73,31 @@ You can set the `tsc` compiler options in the `tsconfig.json` file for the proje
...
 In the HTML code for the extensions, the JavaScript files are referenced. The following example, links to the `datasources.js` file.
 
 ```html
-    <!-- Extensions Library (this will be hosted on a CDN eventually) -->
-    <script src="../../lib/tableau.extensions.1.latest.js"></script>
-
-    <!-- The  extension code (webpack) -->
-    <script src="../../dist/datasources.js"></script>
-
+<!-- Extensions Library (this will be hosted on a CDN eventually) -->
```
Comment (seanmakesgames) [inline docs/trex_typescript.md:90]: This is hosted on our sandboxing server if we want to start giving that url out to customers

**Review by seanmakesgames — APPROVED** [review]
Looks good other than opening up a conversation about cdn

File: docs/trex_typescript.md (hunk 1 above)
Comment (johnDance) [reply docs/trex_typescript.md:90]: Should we put...

How do I find the sandboxing server url?

Comment (johnDance) [reply docs/trex_typescript.md:90]: Markdown removed most of my comment. I'm going to leave it as is this time around.

**Pull request merged**